specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -442,7 +443,7 @@ technology.link:#a3665[18]
 [[a2331]]
 ==== Pruned Java Technologies
 
-As the Jakarta EE specification has evolved,
+As the Jakarta™ EE specification has evolved,
```

Comment:
The (TM) is not required any further in this section.  You caught it at the beginning of the chapter, that's enough.